deps/chakrashim/core/Build/armasm.targets

```diff
@@ -0,0 +1,163 @@
+       FunctionLevelLinking               ="%(ClCompile.FunctionLevelLinking)"
+       GenerateXMLDocumentationFiles      ="%(ClCompile.GenerateXMLDocumentationFiles)"
+       IgnoreStandardIncludePath          ="%(ClCompile.IgnoreStandardIncludePath)"
+       InlineFunctionExpansion            ="%(ClCompile.InlineFunctionExpansion)"
+       IntrinsicFunctions                 ="%(ClCompile.IntrinsicFunctions)"
+       MinimalRebuild                     ="%(ClCompile.MinimalRebuild)"
+       MultiProcessorCompilation          ="%(ClCompile.MultiProcessorCompilation)"
+       ObjectFileName                     ="$(IntDir)"
+       OmitDefaultLibName                 ="%(ClCompile.OmitDefaultLibName)"
+       OmitFramePointers                  ="%(ClCompile.OmitFramePointers)"
+       OpenMPSupport                      ="%(ClCompile.OpenMPSupport)"
+       Optimization                       ="%(ClCompile.Optimization)"
+       PrecompiledHeader                  =""
+       PrecompiledHeaderFile              ="NotUsing"
+       PrecompiledHeaderOutputFile        =""
+       PreprocessKeepComments             ="%(ClCompile.PreprocessKeepComments)"
+       PreprocessorDefinitions            ="%(ClCompile.PreprocessorDefinitions)"
+       PreprocessSuppressLineNumbers      ="%(ClCompile.PreprocessSuppressLineNumbers)"
+       PreprocessToFile                   ="true"
+       ProcessorNumber                    ="%(ClCompile.ProcessorNumber)"
+       ProgramDataBaseFileName            ="%(ClCompile.ProgramDataBaseFileName)"
+       RuntimeLibrary                     ="%(ClCompile.RuntimeLibrary)"
+       RuntimeTypeInfo                    ="%(ClCompile.RuntimeTypeInfo)"
+       ShowIncludes                       ="%(ClCompile.ShowIncludes)"
+       SmallerTypeCheck                   ="%(ClCompile.SmallerTypeCheck)"
+       StringPooling                      ="%(ClCompile.StringPooling)"
+       StructMemberAlignment              ="%(ClCompile.StructMemberAlignment)"
+       SuppressStartupBanner              ="%(ClCompile.SuppressStartupBanner)"
+       TreatSpecificWarningsAsErrors      ="%(ClCompile.TreatSpecificWarningsAsErrors)"
+       TreatWarningAsError                ="%(ClCompile.TreatWarningAsError)"
+       TreatWChar_tAsBuiltInType          ="%(ClCompile.TreatWChar_tAsBuiltInType)"
+       UndefineAllPreprocessorDefinitions ="%(ClCompile.UndefineAllPreprocessorDefinitions)"
+       UndefinePreprocessorDefinitions    ="%(ClCompile.UndefinePreprocessorDefinitions)"
+       UseFullPaths                       ="%(ClCompile.UseFullPaths)"
+       UseUnicodeForAssemblerListing      ="%(ClCompile.UseUnicodeForAssemblerListing)"
+       WarningLevel                       ="%(ClCompile.WarningLevel)"
+       WholeProgramOptimization           ="%(ClCompile.WholeProgramOptimization)"
+       XMLDocumentationFileName           ="%(ClCompile.XMLDocumentationFileName)"
+
+       TrackerLogDirectory                ="%(ClCompile.TrackerLogDirectory)"
```

Comment:
Thansk @thefourtheye  for pointing it out. However since these sources are pulled from [ChakraCore](https://github.com/Microsoft/chakracore) repo, feel free to open an issue on ChakraCore.